origin-docs/source/includes/resources/origin.md

```diff
@@ -18,18 +18,37 @@ let origin = new Origin(configOptions)
  - `ipfsApiPort`
  - `ipfsGatewayPort`
  - `ipfsGatewayProtocol`
+ - `discoveryServerUrl`
+ - `messagingNamespace`
+ - `arbitrator`
+ - `affiliate`
+ - `blockEpoch`
  - `attestationServerUrl`
+ - `ipfsCreator`
+ - `OrbitDB`
+ - `ecies`
+ - `web3`
+ - `contractAddresses`
 
- For example, if you are running a local IPFS daemon then you could set following config options:
+ > For example, if you are running a local IPFS daemon then you could set following config options:
 
 ```javascript
-let origin = new Origin(configOptions)
-
-{
+const configOptions = {
```

Comment:
I'm not sure if it makes sense to distract with all of these `null-ish` options. I think I would prefer to limit this illustration to just the ipfs-related options previously included. For example, passing those values for `arbitrator` and `affiliate` would break things, but I don't think we need to put the appropriate values here in the docs.